### ISSUE TITLE
NTR: Acetoacetate low in blood inappropriately (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8321]
New term request Acetoacetate low in blood inappropriately (elthat_130306175914): Ketone; acetoacetic acid

**Comment by pnrobinson** [comment]
@MickeySegal

Is this the phenotype you are referring to?
def: Lower than appropriate concentration of acetoacetate in states such as hypoglycemia in which the normal response would be to elevate acetoacetate.
Comment: Ketone synthesis in the liver produces acetoacetate and beta-hydroxybutyrate from two acetyl CoA molecules. This occurs during ketoacidosis in physiological (starvation) and disease-associated (diabetic ketoacidosis) states.

**Comment by MickeySegal** [comment]
Yes.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000235
- label: Inappropriately low concentration of acetoacetate
- parents: [Abnormal homeostasis (HP:0012337)](https://hpo.jax.org/app/browse/term/HP:0012337)
- pmid: 
- def: Lower than appropriate concentration of acetoacetate in states such as hypoglycemia in which the normal response would be to elevate acetoacetate.
- comment: Ketone synthesis in the liver produces acetoacetate and beta-hydroxybutyrate from two acetyl CoA molecules. This occurs during ketoacidosis in physiological (starvation) and disease-associated (diabetic ketoacidosis) states.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8320
- synonyms:
